test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1202,11 +1202,16 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 					"hugepages-1Gi":       resource.MustParse("1Gi"),
 				},
 			},
+			// the free resources that should be left on the target node should not depend that there will be some baseload added upon padding the node,
+			// those free resources should match the pod requests in total. The reason behind that is that Noderesourcesfit plugin (the plugin that is
+			// responsible for accepting/rejecting compute nodes as candidates for placing the pod) actually accounts for the baseload, it compares the
+			// actual available resources on node with the pod requested resources, if the available resources can accommodate the pod resources then it
+			// will mark the node as a possible candidate, if not it will reject it.
```

Comment:
note to (my)self: `Fit` does sum all the container resources requests; hence, will account for the baseload when running it's own computation. (note baseload does the same thing/uses the same algo - actually inspired from that).